DEV-671 Side Navigation redesign with light/dark mode

willmccormick24 opened the pull request:
Updated side navigation with the redesign. Put IBC Transfer under Tools, updated external links, icons, and layout. Instead of turning all the SVGs into components I inverted` <img>`tag based on theme. 

Light Mode (signed in):
![Screenshot 2023-09-27 at 3 20 39 PM](https://github.com/rebuschain/rebus.app.web/assets/143646590/4e5fbadb-3f09-464c-afcf-6873325d7ba9)

Dark mode (signed out - tools opened):
![Screenshot 2023-09-27 at 3 21 49 PM](https://github.com/rebuschain/rebus.app.web/assets/143646590/ee5bd946-b889-4d6a-9b44-ac7b9a136cb1)

willmccormick24 commented:
Oh one more thing: I was uncertain what the first external link was for on the Figma design. It was a globe icon marked as "language" but does Rebus app support multiple languages? I kept the old Medium icon/link in the meantime

nicolam1 reviewed (dismissed):
LGTM

nicolam1 commented:
@willmccormick24 the PR looks good but I don't see the nav for EVM and Explorer 
![Screenshot 2023-09-28 at 10 28 56 AM](https://github.com/rebuschain/rebus.app.web/assets/1234418/25412849-fa6e-4de7-b879-a3518fc8528f)
Can we add those as well?

willmccormick24 commented:
> @willmccormick24 the PR looks good but I don't see the nav for EVM and Explorer ![Screenshot 2023-09-28 at 10 28 56 AM](https://user-images.githubusercontent.com/1234418/271318574-25412849-fa6e-4de7-b879-a3518fc8528f.png) Can we add those as well?

@nicolam1 Sure no problem I can add those. I was going to add them later once Cosmos and EVM reskins were complete, but I'll update it now. Clicking on them should trigger external links to Explorer and EVM in the same window right?

nicolam1 commented:
@willmccormick24 you can use a feature flag to hide those menu items while we skin the other apps. Also please make the main URL they point to configurable so we can have different URLs for testnet and mainnet.

Ericky14 commented on src/components/layouts/sidebar/index.tsx:
```diff
@@ -42,90 +43,98 @@ export const Sidebar: FunctionComponent<React.PropsWithChildren<unknown>> = obse
-												return featureFlagStore.featureFlags.assetsPage;
-											}
+		<SideBarStyled>
+			<React.Fragment>
```
Remove React.Fragment if you're wrapping it

Ericky14 reviewed (changes requested):
There's something weird going on in the sidebar, when you change routes the text blinks white

![Oct-02-2023 11-17-05](https://github.com/rebuschain/rebus.app.web/assets/21248862/7af71a50-d991-4f03-aaba-cb7e4d19cef6)

willmccormick24 commented:
Added EVM/Explorer with a feature flag and configurable links. Also addressed Ericky's comments

![Screenshot 2023-10-02 at 12 21 43 PM](https://github.com/rebuschain/rebus.app.web/assets/143646590/656c6acf-c7ed-4e23-9591-39588dfb67c8)

nicolam1 reviewed (approved):
LGTM
